parser_library/test/lsp/lsp_context_ordinary_operands_test.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2023 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+#include "../common_testing.h"
```

Comment:
```suggestion
#include <utility>

#include "../common_testing.h"
```

Reply:
utility is included in the common_testing header.